src/libretro/FrontendBridge.cpp

```diff
@@ -155,16 +155,18 @@ int16_t CFrontendBridge::InputState(unsigned int port, unsigned int device, unsi
       {
         if (id == RETRO_DEVICE_ID_ANALOG_X)
         {
-          const float normalized = (x + 1.0f) / 2.0f;
-          inputState = (int)(normalized * 0xffff) - 0x8000;
+          value = x;
         }
         else if (id == RETRO_DEVICE_ID_ANALOG_Y)
         {
-          const float normalized = (-y + 1.0f) / 2.0f; // y axis is inverted
-          inputState = (int)(normalized * 0xffff) - 0x8000;
+          value = -y; // y axis is inverted
         }
       }
     }
+
+    const float normalized = (value + 1.0f) / 2.0f;
+    const int clamped = std::max(0, std::min(0xffff, static_cast<int>(normalized * 0xffff)));
```

Comment:
`#include <algorithm>` is missing for `std::max` & `std::min`

Reply:
lol always forget that max and min are "algorithms". pushed a fix, thanks